docfx_project/documentation/Page.AdjustCommonSettings.md

```diff
@@ -0,0 +1,172 @@
+# How to adjust some common settings, including set BrowsersPath and DriverExecutablePath but not limited
+_Contributors: [Scott Huang](https://github.com/ScottHuangZL)_
+
+## Problem
+
+It is not easy or not convenient to let end user to: 
+
+    install nodejs firstly 
```

Comment:
What do you mean with this?

Reply:
I complete build the application and copy the published folder to a machine(Win server 2016) which no .netcore & no nodejs, and run "playwright-cli install" it will show error as below:
So, I assume playwright-cli may be depends on something, nodejs?

I can remove this wording if you sure playwrght-cli is independent.  

```
(node:10220) UnhandledPromiseRejectionWarning: Error: read ECONNRESET
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:205:27)
(node:10220) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:10220) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Reply:
That sounds like a network error. I don't think that's related to dependencies.

Reply:
I have remove that related wording "install nodejs firstly " more than 1 weeks ago.